byteflow2.py

```diff
@@ -740,7 +740,7 @@ def foo(n):
     for_iter: Label = next(iter(headers))
     # Create the SyntheticForIter block and replicate the jump targets from the
     # original FOR_ITER block.
-    synth_for_iter = SyntheticForIter(bbmap.clg.new_index())
+    synth_for_iter = SyntheticForIter(str(bbmap.clg.new_index()))
```

Comment:
Note to self, this is ambiguous throught the code base and needs to be revisited, I think the indices should be integers, not strings. But part of the test framework uses strings, so there is some refactoring to be done here.